pulpcore/cli/common/context.py

```diff
@@ -279,6 +279,15 @@ def pulp_href(self, value: str) -> None:
         self._entity_lookup = {"pulp_href": value}
         self._entity = None
 
+    @property
+    def field_value(self) -> Any:
+        """Property to return the value passed into entity"""
+        if self._entity_lookup and "name" in self._entity_lookup.keys():
+            return self._entity_lookup["name"]
+        raise click.ClickException(
+            "field_value should only be access after a context is created with entity data"
```

Comment:
@mdellweg Here is something I thought of when discussing the earlier issue with naming that hideous type. Having the repository callbacks return the context and then using this property inside the command methods to determine if the user passed in nothing, "", or a lookup-able name

Reply:
I don't see yet, how this is meant to make it better, and it certainly does not comply with the way you use it here: https://github.com/pulp/pulp-cli/pull/139/files#diff-1c220566888eccc231addf457caa58d092678351bf0317388a64b4f92d4df749R103

Reply:
Functionally it does nothing new, it just removes the need for an additional type. It's probably not the best to do this, but having the callbacks return three possible types also doesn't seem right either. It might be best to implement this as another parameter when declaring a context. Let me revise it.

Reply:
Anyway, this property being soulbound to working with `"name"` doesn't look right to me.
The returning of three different things is only like "that thing or no thing or delete it". I think that is well in the python mindset of "something or None".
Sadly i figured, that python 3.5 does not have the `Literal` type, so the type allows for every string.

Reply:
I agree, removing it and going with EntityFieldDefinition typing.